Frontend/ui-common/src/Components/ChatView/Controller.purs

```diff
@@ -44,18 +45,8 @@ type Config =
   , showStroke :: Boolean
   , languageKey :: String
   , mapsText :: String
-  , grey700 :: String
```

Comment:
@PraveenGongada Why was this added ? 
Can you review this if it is finer removing

Reply:
These were added because then we don't have common colors.purs file, now they can be removed